src/monodroid/jni/debug.cc

```diff
@@ -474,7 +474,7 @@ Debug::process_cmd (int fd, char *cmd)
 		log_info (LOG_DEFAULT, "Debugger requested an exit, will exit immediately.\n");
 		fflush (stdout);
 		fflush (stderr);
-		exit (0);
+		Helpers::abort_application ();
```

Comment:
This gives me pause; the original `exit(0)` is not signaling an *error* condition, so is calling `abort_application()` appropriate?

As this is part of the debugging infrastructure, I think it may be safer to keep this as `exit(0)`, unless you want to do an end-to-end test to exercise this behavior and see what happens…

Reply:
The problem here is that the intention is to terminate the application
but `exit` doesn't do that while `abort` does.

From the pov of the debugger the application dies anyway
but from the pov of the developer, it will immediately restart itself
which is rather weird